Explorer++/Explorer++/Explorer++.cpp

```diff
@@ -77,6 +77,7 @@ m_hContainer(hwnd)
 	m_hActiveListView				= NULL;
 	m_hTabFont						= NULL;
 	m_hNextClipboardViewer			= NULL;
+	m_hLanguageModule				= NULL;

```

Comment:
I had a crash without this line so I'll leave it here as well although it has nothing common with the problem in the topic.